sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,10 +1,20 @@
-## Release History
+# Release History
```

Comment:
I believe these header changes will break some of the cosmos docs which directly import this file so we will likely need to undo that. 

FYI @kushagraThapar

Reply:
LOL nevermind it looks like @kushagraThapar  already caught this in the review. I think we will have to manually undo those changes as the changelog automation does not currently support this scenario.

Reply:
@weshaggard - will we need to do this every single time going forward ?
Can we skip azure-cosmos and azure-spring-data-cosmos changelog formatting in some way ?

Reply:
@chidozieononiwu can you please file a tracking issue to try and see what we can do to handle this custom formatting problem.

@kushagraThapar we should follow back up with the docs folks to see if we can get them to fix this as I still do not like the way they are directly consuming this.

Reply:
Sure, we can follow up on that!
ccing - @SnehaGunda

Reply:
Filed https://github.com/Azure/azure-sdk-for-java/issues/23908 to track the changelog automation issue.